files/helpers.py

```diff
@@ -687,8 +690,7 @@ def produce_ffmpeg_commands(media_file, media_info, resolution, codec, output_fi
     #        target_fps = 25
     #    else:
 
-    # adjust the target frame rate if the input is fractional
-    target_fps = src_framerate if isinstance(src_framerate, int) else math.ceil(src_framerate)
+    target_fps = src_framerate
```

Comment:
This does not make sense if you override the `target_fps` in line 506, no? It will always be reset to the source framerate.

In principle I think keeping it exactly the same as the source framerate is meaningful provided the source framerate can be parsed correctly even if it's a fractional framerate.

The other alternative might be to not specify the framerate at all, and let ffmpeg encode it as-is. Just capture cases of too high framerates and scale them down.

Reply:
Yes, I guess, I just did not wanted to break the naming of variables. The framerate should be kept the same as original except the case you want to provide <=30Hz encodes for videos with >30Hz (like YouTube does - and I think it does not make sense to have 240p60, but that should be left to user to decide in encoding profiles) and case when the framerate of original is >60Hz.